cronback-lib/database/mod.rs

```diff
@@ -2,32 +2,21 @@ mod errors;
 mod pagination;
 
 pub use errors::DatabaseError;
-use migration::{Migrator, MigratorTrait};
 pub use pagination::*;
-use sea_orm::TransactionTrait;
 
 #[derive(Clone)]
 pub struct Database {
     pub orm: sea_orm::DatabaseConnection,
 }
 
 impl Database {
-    pub async fn connect(conn_string: &str) -> Result<Self, sea_orm::DbErr> {
+    pub async fn connect(conn_string: &str) -> Result<Self, DatabaseError> {
         Ok(Self {
             orm: sea_orm::Database::connect(conn_string).await?,
         })
     }
 
-    pub async fn in_memory() -> Result<Self, sea_orm::DbErr> {
-        let conn = Self::connect("sqlite::memory:").await?;
-        conn.migrate().await?;
-        Ok(conn)
-    }
-
-    pub async fn migrate(&self) -> Result<(), sea_orm::DbErr> {
-        let conn = self.orm.begin().await?;
-        Migrator::up(&conn, None).await?;
-        conn.commit().await?;
-        Ok(())
```

Comment:
Did templating it not work?

```
pub async fn migrate<T: MigratorTrait>(&self) -> Result<(), sea_orm::DbErr>
```

Reply:
This will be removed in a future PR, it's a temporary measure for now.